src/components/OtherHero/index.js

```diff
@@ -0,0 +1,160 @@
+import RichTypography from "@commons-ui/core/RichTypography";
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import mapLines from "@/pesayetu/assets/images/Mask Group 8@2x.png";
+import Section from "@/pesayetu/components/Section";
+
+const useStyles = makeStyles(({ palette, typography, breakpoints }) => ({
+  root: {
+    backgroundColor: palette.background.paper,
+    backgroundPosition: "50% 100%",
+    minHeight: "min-content",
+    position: "relative",
+    paddingBottom: typography.pxToRem(43),
+    [breakpoints.up("md")]: {
+      paddingBottom: typography.pxToRem(98),
+    },
+  },
+  textContainer: {
+    height: "100%",
+    paddingTop: typography.pxToRem(50),
+    [breakpoints.up("lg")]: {
+      paddingTop: typography.pxToRem(129.94),
+    },
+  },
+  intro: {
+    fontWeight: "bold",
+  },
+  title: {
+    marginBottom: typography.pxToRem(20),
+    fontWeight: "bold",
+    [breakpoints.up("md")]: {
+      marginBottom: typography.pxToRem(40),
```

Comment:
`typography.pxToRem(35.5)`? As per design?

Reply:
no it's 40 @gertie-sheshe 

<img width="500" src="https://user-images.githubusercontent.com/23483005/127786091-8c995d12-6460-4b17-a435-70184d639870.png"/>

Reply:
@gertie-sheshe

Reply:
Ah, yeah you are right 💯 I was seeing it this way (for desktop):
<img width="421" alt="Screenshot 2021-08-02 at 15 41 26" src="https://user-images.githubusercontent.com/12892109/127864328-c956d89c-8354-45b3-a5c8-a37733afb6c6.png">


Also, the highlighted background is still missing, and the `marginBottom` for desktop is 20 and not 40:
<img width="529" alt="Screenshot 2021-08-02 at 15 48 03" src="https://user-images.githubusercontent.com/12892109/127864528-1aa54c27-5fab-4903-b668-cffe27c6a66e.png">